packages/sdk/src/utils/BrandKitHelper.ts

```diff
@@ -0,0 +1,163 @@
+import {
+    APIBrandKit,
+    BrandKitCharacterStyle,
+    BrandKitColor,
+    BrandKitInternal,
+    BrandKitParagraphStyle,
+    HEXColorValue,
+    SpotCMYKColorValue,
+    SpotHEXColorValue,
+    SpotRGBColorValue,
+} from '../types/BrandKitTypes';
+import { CharacterStyleUpdate } from '../types/CharacterStyleTypes';
+import {
+    APIColorType,
+    Color,
+    ColorType,
+    ColorUpdate,
+    ColorUsageType,
+    DocumentColor,
+    SpotColorCMYK,
+    SpotColorHEX,
+    SpotColorRGB,
+} from '../types/ColorStyleTypes';
+import { DocumentFontFamily } from '../types/FontTypes';
+import { ParagraphStyleUpdate } from '../types/ParagraphStyleTypes';
+
+const isSpotHexColor = (color: Color): color is SpotColorHEX => color.type === ColorType.spotHEX;
+const isSpotRgbColor = (color: Color): color is SpotColorRGB => color.type === ColorType.spotRGB;
+const isSpotCmykColor = (color: Color): color is SpotColorCMYK => color.type === ColorType.spotCMYK;
+
+const isSpotColor = (color: Color): color is SpotColorRGB | SpotColorCMYK | SpotColorHEX =>
+    isSpotHexColor(color) || isSpotRgbColor(color) || isSpotCmykColor(color);
+
+const isBranKitSpotHexColor = (color: BrandKitColor): color is SpotHEXColorValue => color.type === APIColorType.spotHex;
+const isBranKitHexColor = (color: BrandKitColor): color is HEXColorValue => color.type === APIColorType.hex;
```

Comment:
Small spelling mistakes

```suggestion
const isBrandKitSpotHexColor = (color: BrandKitColor): color is SpotHEXColorValue => color.type === APIColorType.spotHex;
const isBrandKitHexColor = (color: BrandKitColor): color is HEXColorValue => color.type === APIColorType.hex;
```